test/utils/test_pulse_scaling.py

```diff
@@ -138,3 +138,13 @@ def test_forced_rzz_template_match(self):
                 Operator(scale_qc_match.bind_parameters({theta: theta_set}))
             )
         )
+
+    def test_angle_reduction(self):
+        """Test Angle Reduction"""
+        backend = FakeMumbai()
+
+        qc = QuantumCircuit(2)
+        qc.rzx(9 * np.pi / 2, 0, 1)
+        qc_s = scale_cr_pulses(qc, backend, param_bind={})
```

Comment:
I meant something like this, to directly test the pass:
```python
pass_manager = PassManager(ReduceAngles(["rzx"]))
qc_s = pass_manager.run(qc)
```

Reply:
Ah yes I see, rewrote to just use the single pass.